dandi/tests/test_download.py

```diff
@@ -88,7 +100,7 @@ def test_download_000027_metadata_only(url: str, tmp_path: Path) -> None:
 @mark.skipif_no_network
 @pytest.mark.parametrize(
     "url",
-    [  # Should go through API
+    [  # Should go through AP
```

Comment:
what is AP?

Reply:
I did not mean to make that change, and I don't know how it happened.

Reply:
ok, pushed 10d076e7bd9625840d8a4e1cab8a49d986c415ea directly to master to revert this one